test/Ocelot.IntegrationTests/ocelot.json

```diff
@@ -33,7 +33,7 @@
         "ClientWhitelist": [],
         "EnableRateLimiting": false,
         "Period": null,
-        "PeriodTimespan": 0.0,
+        "PeriodTimespan": 0,

```

Comment:
What's the problem with `0.0`?
Have some integration tests failed because of `0.0` constant?

Reply:
I've run integration tests locally and they've passed, which matches the current state in the develop branch. Therefore, I don't understand the reason for this change.

Reply:
Reverted in https://github.com/ThreeMammals/Ocelot/pull/2125/commits/2ffd8a00f4dbb1f8bdde83f38e67fd70e7e96c4b

Reply:
This change is because system text json behaves like this.